src/flow/src/expr/error.rs

```diff
@@ -117,6 +117,7 @@ pub enum EvalError {
 
     #[snafu(display("DataFusion error: {raw:?}, context: {context}"))]
     Datafusion {
+        #[snafu(source)]
         raw: DataFusionError,
```

Comment:
Otherwise it cannot be recognize by our macro

```suggestion
        error: DataFusionError,
```